src/TLuaInterpreter.cpp

```diff
@@ -4659,6 +4659,14 @@ int TLuaInterpreter::getMudletHomeDir( lua_State * L )
     return 1;
 }
 
+int TLuaInterpreter::getMudletLuaDefaultPath( lua_State * L )
+{
+    QString path = LUA_DEFAULT_PATH "/";
```

Comment:
The reason I keep going on about `QStringLiteral(`...`)` is that once `QT_NO_CAST_FROM_ASCII` and `QT_NO_CAST_TO_ASCII` are `#defined` - all raw "strings" used to initialise/compare with QStrings will generate a compile error - and THIS file is a particularly bad case with over a thousand cases at the moment!.  This is a powerful step in order to find the strings that need to be wrapped in `tr(`...`)` so that they can be translated to support more than the EN-us locale - and the `QStringLiteral(`...`)` wrapper is the easiest way to wrap strings that are NOT to be so translated.  Note that some MSVC versions fail to handle QStringLiteral( "stringa" "stringb" ) which is why with a mixture of constant and semi-variable constructs should use the `.arg( variable )` variable replacement construct, i.e.:
> QString path = QStringLiteral( "%1/" ).arg( LUA_DEFAULT_PATH )